apps/mobile/fastlane/Matchfile

```diff
@@ -0,0 +1,13 @@
+git_url("https://github.com/leather-io/mobile-signing-certificates")
+
+storage_mode("git")
+
+type("appstore") # The default type, can be: appstore, adhoc, enterprise or development
+
+app_identifier(["io.leather.mobilewallet"])
+# username("user@fastlane.tools") # Your Apple Developer Portal username
```

Comment:
Maybe delete all the commented out stuff now so it's clear what the exact config is?

Reply:
Done